docs/basic-features/seo-and-meta-manager.md

```diff
@@ -50,7 +40,72 @@ render() {
 }
 ```
 
-### Configuring meta tags - `setMetaName()`, `getMetaName()`, `getMetaNames()`
+## Route-specific meta tags
+
+The following snippet is included in your `DocumentView` by default. It dynamically renders your page-specific meta tags
+and title described in `setMetaParams` controller method.
+
+```jsx
+// app/component/document/DocumentView.jsx`
+
+// in the beginning of render() method
+const { metaManager } = this.props;
+
+// ...
+<head>
+  {metaManager.getMetaNames().map(name => (
+    <meta
+      key={name}
+      name={name}
+      {...metaManager.getMetaName(name)}
+      data-ima-meta
+    />
+  ))}
+  {metaManager.getMetaProperties().map(property => (
+    <meta
+      key={property}
+      property={property}
+      {...metaManager.getMetaProperty(property)}
+      data-ima-meta
+    />
+  ))}
+  {metaManager.getLinks().map(rel => (
+    <link
+      key={rel}
+      rel={rel}
+      {...metaManager.getLink(rel)}
+      data-ima-meta
+    />
```

Comment:
Why do we create component for that?

`<Meta metamanager={metamanager}/>`

Reply:
We could also do this in server page renderer with the new content interpolation like runner and styles

Reply:
Yes, It can be.